lib/generators/default_views/USAGE

```diff
@@ -0,0 +1,18 @@
+Description:
```

Comment:
Is this USAGE file special? Or just the place you chose to keep these notes?

I'd definitely suggest adding this to the README at the top of the repository.